app/views/layouts/application.html.haml

```diff
@@ -12,6 +12,7 @@
     %meta{ name: "language-server-url", content: Exercism.config.language_server_url }
     - if user_signed_in?
       %meta{ name: "bug-reports-url", content: Exercism::Routes.api_bug_reports_url(auth_token: current_user.auth_tokens.first.to_s) }
+      %meta{ name: "parse-markdown-url", content: Exercism::Routes.api_parse_markdown_url }
```

Comment:
I don't think any of these need to rely on `user_signed_in`. Let's remove that check, and just always render them in head. Makes it easier to consistently cache <head> when I add caching.